lib/PuppeteerSharp/Page.cs

```diff
@@ -63,28 +76,97 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
             Client.MessageReceived += client_MessageReceived;
         }
 
+        internal Session Client { get; }
+
         #region Public Properties
+
+        /// <summary>
+        /// Emitted when the JavaScript <c>load</c> <see cref="https://developer.mozilla.org/en-US/docs/Web/Events/load"/> event is dispatched.
+        /// </summary>
         public event EventHandler<EventArgs> Load;
+
+        /// <summary>
+        /// Emitted when the page crashes
+        /// </summary>
         public event EventHandler<ErrorEventArgs> Error;
+
+        /// <summary>
+        /// Emitted when the JavaScript code makes a call to <c>console.timeStamp</c>. For the list of metrics see <see cref="Page.MetricsAsync"/>.
+        /// </summary>
         public event EventHandler<MetricEventArgs> MetricsReceived;
```

Comment:
I think that this should be renamed to `Metrics` to match puppeteer

Reply:
Same